docs/platform/howto/create-service-integration.rst

```diff
@@ -0,0 +1,32 @@
+Create a service integration
+============================
+
+If you're not sure what a service integration is, see :doc:`this guide <../concepts/service-integration>` for an overview. This article will guide you on how to create service integrations between different Aiven services and move telemetry data using these integrations. To get started, you will need three services:
+
+- The service that produces the telemetry data. We're using an Aiven for Apache Kafka® service in our example, but this could be one of the other Aiven services available.
+- Aiven for PostgreSQL® - Database where the telemetry data is stored and can be queried from
+- Aiven for Grafana® - Dashboards for the telemetry data
+
+.. note::
+
+    Currently these services need to be running in the same Aiven project, but this limitation will be removed in the future and you will be able to, for example, connect multiple Aiven for Apache Kafka® services from different projects to the same PostgreSQL® and Grafana® instances.
```

Comment:
In the Description for this PR, you say that this note has been removed.
Should we remove it?
I assume that users will expect that services need to be in the same project, so I don't think it's worth keeping...